# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qscanq/loguser.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/conf-users.orig	2005-11-28 21:26:37.000000000 -0300
+++ ./src/conf-users	2005-11-28 21:26:44.000000000 -0300
@@ -1,5 +1,5 @@
 Gqscanq
-Gqscanlog
+log
 
 The first line of this file specifies the user ID that will own the virus
 scanning queue, and as which qscanq will run. It should be a user
